src/main/scala/cromwell/util/google/GoogleCredentialFactory.scala

```diff
@@ -96,11 +96,15 @@ abstract class GoogleCredentialFactory {
   }
 
   private def forServiceAccount(serviceAccountMode: ServiceAccountMode): Credential = {
+    val pemFile = new File(serviceAccountMode.pemPath)
+    if (!pemFile.exists()) {
+      throw new FileNotFoundException(s"Pem file ${Paths.get(serviceAccountMode.pemPath).toAbsolutePath} does not exist")
+    }
```

Comment:
No when I was trying to write a test for preemption on JES I ran into some not very clear exceptions so I added some more context to some of them.